products/rhel10/profiles/stig_gui.profile

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - mab879
+
+
+reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
+
+title: 'DRAFT - DISA STIG for Red Hat Enterprise Linux 10'
+
+description: |-
+    This is a draft profile based on its RHEL8 version for experimental purposes.
+    It is not based on the DISA STIG for RHEL9, because this one was not available at time of
```

Comment:
```suggestion
    This is a draft profile based on its RHEL9 version for experimental purposes.
    It is not based on the DISA STIG for RHEL10, because this one was not available at time of
```